src/login/components/UserProfileFormFields.svelte

```
@@ -0,0 +1,207 @@
+  });
+  onMount(() => {
+    const unsubscribe = formState.subscribe(({ isFormSubmittable }) => {
+      onIsFormSubmittableValueChange(isFormSubmittable);
+    });
+    return () => unsubscribe();
+  });
+  const placeholderMap: Record<string, string> = {
+    firstName: 'Jack',
+    lastName: 'Albert',
+    email: 'example@email.com',
+    password: 'Create your password'
```

ESLint

[error] Insert `,`

(prettier/prettier)